src/Interactive/EditorFeatures/Core/Extensibility/Interactive/ResetInteractive.cs

```diff
@@ -104,11 +104,12 @@ internal Task Execute(IInteractiveWindow interactiveWindow, string title)
                 await interactiveEvaluator.SetPathsAsync(referenceSearchPaths, sourceSearchPaths, projectDirectory).ConfigureAwait(true);
             }
 
-            await interactiveWindow.SubmitAsync(new[]
+            var importReferencesCommand = referencePaths.Select(_createReference);
```

Comment:
Is there a `#r` command that permits extern aliases?
At the moment all references get `#r`ed. This means that it will not be possible to access ambiguious objects.